### PR TITLE
Added and Updated HTML.md

### PR DESCRIPTION
**Pull request by keiwsh**
As described. Tried to match the "layout" from already existing docs otherwise open to feedback.

### CODE REVIEW

**Review by JDSherbert — APPROVED** [review]
lgtm

**Comment by JDSherbert** [comment]
@VerzatileDev just waiting for your sign-off dude

File: docs/Language/HTML/HTML.md
Comment (VerzatileDev) [inline docs/Language/HTML/HTML.md:11]: I would consider removing the following ` ## What is HTML?  ` as the ` # {{ page.title }} ` would let the user know what it is about "HTML " and the describtion written states what the following is.

File: docs/Language/HTML/HTML.md
Comment (VerzatileDev) [inline docs/Language/HTML/HTML.md:17]: Change the following ` 1. **Markup Language** ` to ` 1. **Markup Language: ** `  Referencing a list of the following.  

- Using " : "

**Review by VerzatileDev — CHANGES_REQUESTED** [review]
Well written page, thank you for the contribution though please see the small nit-picking areas as changes.

Otherwise approved.

**Comment by keiwsh** [comment]
Did the changes. Thanks for the feedback. :smiley:

**Review by VerzatileDev — APPROVED** [review]
Lgtm!

**Pull request merged**
